darts/models/forecasting/pl_forecasting_module.py

```diff
@@ -100,6 +108,23 @@ def __init__(
             dict() if lr_scheduler_kwargs is None else lr_scheduler_kwargs
         )
 
+        if torch_metrics:
+            try:
+                if isinstance(torch_metrics, torchmetrics.Metric):
+                    torch_metrics = torchmetrics.MetricCollection([torch_metrics])
+                self.train_metrics = torch_metrics.clone(prefix="train_")
+                self.val_metrics = torch_metrics.clone(prefix="val_")
+            except AttributeError:
+                raise_log(
+                    AttributeError(
+                        "torch_metrics only accepts type torchmetrics.Metric or torchmetrics.MetricCollection"
+                    ),
+                    logger,
+                )
+        else:
+            self.train_metrics = torchmetrics.MetricCollection([])
+            self.val_metrics = torchmetrics.MetricCollection([])
```

Comment:
Let's avoid try/except with something like below.

```suggestion
        if torch_metrics is None:
            torch_metrics = torchmetrics.MetricCollection([])
        elif isinstance(torch_metrics, torchmetrics.Metric):
            torch_metrics = torchmetrics.MetricCollection([torch_metrics])
        elif isinstance(torch_metrics, torchmetrics.MetricCollection):
            pass
        else:
            raise_log(
                AttributeError(
                    "`torch_metrics` only accepts type torchmetrics.Metric or torchmetrics.MetricCollection"
                ),
                logger,
            )
        self.train_metrics = torch_metrics.clone(prefix="train_")
        self.val_metrics = torch_metrics.clone(prefix="val_")
```